src/overlays/actors/ovl_Bg_Treemouth/z_bg_treemouth_cutscene_data.c

```diff
@@ -0,0 +1,93 @@
+#include "z_bg_treemouth.h"
+
+u32 D_808BCE20[] = {
+    0x0000000C, 0x00000BB8, 0x00000015, 0x00000001, 0x00010000, 0x0BB80000, 0x00000000, 0x00000000, 0x0000000F,
```

Comment:
not sure if we want to start using tharos tool to generate macros for cutscene data moving forward. given that its in the repo for other things I think we probably should but want to hear what others think.

regardless of that, the type is CutsceneData for these

Reply:
Yeah I left these un-converted since I don't know how to do so and am not sure if that's in the scope of decompiling the actor. But if someone educates me how, I'm happy to try. I left the type as u32 since it wasn't decompiled yet.

Reply:
yea in general we want to stop giving things type u32 if we dont know what it is. UNK_TYPE should always be your default. but in this case it is definitely CutsceneData

Reply:
Okay, I changed the types. I'll ask in discord about using the macros.